src/pages/billing/List.js

```diff
@@ -286,9 +286,11 @@ export const BillingList = _.flow(
             this.loadProjects()
           }
         }),
-        noBillingProjects && hasFreeCredits && freeCreditsMessage,
-        noBillingProjects && !hasFreeCredits && noBillingMessage,
-        hasBillingProjects && h(ProjectDetail, { key: selectedName, project: _.find({ projectName: selectedName }, billingProjects) }),
+        Utils.cond(
```

Comment:
@zarsky-broad let me know if there are ways to reduce the `cond` further

Reply:
Maybe something like this?
```js
!!selectedName && Utils.cond(
  [!!billingProjects, () => h(ProjectDetail, { key: selectedName, project: _.find({ projectName: selectedName }, billingProjects) })],
  [!!hasFreeCredits, () => freeCreditsMessage],
  () => noBillingMessage
)
```